src/components/GraphFilter/GraphRefresh.tsx

```diff
@@ -13,13 +13,13 @@ type GraphRefreshProps = {
   options: {
     [interval: number]: string;
   };
-  graphDuration: Duration;
+  graphDuration: DurationIntervalInSeconds;
   onUpdateGraphDuration: (duration: number) => void;
   disabled: boolean;
 };
 
 const GraphRefresh: React.SFC<GraphRefreshProps> = props => {
-  const INTERVAL_DURATION = config().toolbar.intervalDuration;
+  const INTERVAL_DURATION_LIST = config().toolbar.intervalDuration;
```

Comment:
OK, this reinforces my previous comment about changing instances of "duration interval" to just "duration" because here we now have "interval duration" :)  This should probably be renamed to "refresh interval".